orttraining/orttraining/python/ort_trainer.py

```diff
@@ -637,7 +643,9 @@ def _init_session(self):
                 self.gradient_accumulation_steps, bind_parameters=False,
                 use_mixed_precision=self.use_mixed_precision, allreduce_post_accumulation=self.allreduce_post_accumulation_,
                 loss_scale_input_name=self.loss_scale_input_name, scaled_loss_output_name=self.scaled_loss_output_name,
-                partition_optimizer=self.partition_optimizer_, enable_grad_norm_clip=self.enable_grad_norm_clip_,
+                partition_optimizer=self.partition_optimizer_, 
+                seed=self.seed_,
```

Comment:
should we also set the same seed into pytorch (e.g. `torch.manual_seed(seed)`) to make sure frontend and backend have matching seeds?

Reply:
the underline mechanism between ort and pytorch are different. To have the same seed between the two does not guarantee the same number (almost surely they are different).